### PR TITLE
py-htgettoken: add v2.0-2

### PR DESCRIPTION
**Pull request by rahmans1**
Add version 2.0.-2

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-htgettoken/package.py [hunk 1]
```diff
@@ -26,6 +26,8 @@ class PyHtgettoken(PythonPackage):
     # they are special reproducible version numbers from `git describe`
     version("1.16-33-g3788bb4", commit="3788bb4733e5e8f856cee51566df9a36cbfe097d")
     version("1.16-20-g8b72f48", commit="8b72f4800ef99923dac99dbe0756a26266a27886")
+    version("2.0-2", commit="e777edb54eb28194351aee0a818aa70489017ab0")
```
Comment (wdconinc) [inline var/spack/repos/builtin/packages/py-htgettoken/package.py]: Newer version are listed earlier.

**Review by wdconinc — COMMENTED** [review]
~~This package doesn't use gssapi anymore, but switched to kerberos. See e.g. https://github.com/fermitools/htgettoken/releases/tag/v2.0.~~

**Comment by rahmans1** [comment]
@spackbot fix style

**Comment by rahmans1** [comment]
Checked that it works on codespace
```
root@codespaces-b33d8c:/workspaces/spack# spack install --fresh py-htgettoken
root@codespaces-b33d8c:/workspaces/spack# spack load py-htgettoken
root@codespaces-b33d8c:/workspaces/spack# htgettoken --version
2.0
```

File: var/spack/repos/builtin/packages/py-htgettoken/package.py [hunk 2]
```diff
@@ -24,8 +24,10 @@ class PyHtgettoken(PythonPackage):
 
     # The following versions refer to setuptools-buildable commits after 1.16;
     # they are special reproducible version numbers from `git describe`
+    version("2.0-2", commit="e777edb54eb28194351aee0a818aa70489017ab0")
```
Comment (wdconinc) [inline var/spack/repos/builtin/packages/py-htgettoken/package.py]: The comment refers to the 1.16 versions only.
```suggestion
    version("2.0-2", commit="e777edb54eb28194351aee0a818aa70489017ab0")
    # The following versions refer to setuptools-buildable commits after 1.16;
    # they are special reproducible version numbers from `git describe`
```

**Review by wdconinc — APPROVED** [review]
Looks good!

**Pull request merged**